frontend/src/components/aboutMe.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './aboutMe.css';
+import polaroid from '..\\src\\images\\Pizza-Night-Polaroid.jpg'
```

Comment:
I haven't seen this way of importing an image. It can be accessed by changing the link to '../images/Pizza-Night-Polaroid.jpg'